### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by icidasset** [icidasset/icidasset#2]
Things missing:
- Document language (`<html lang="en">`)

Things to check:
- "Color contrast should at least pass WCAG AA (AAA for mobile)"
- "All pages were tested in a screen-reader (VoiceOver, ChromeVox, NVDA or Lynx)"
